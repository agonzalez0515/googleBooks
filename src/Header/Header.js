import React from 'react'
import styles from './header.module.css'

const Header = () => {
  return (
    <header className={styles.header}>
      <h1 className={styles.headerTitle}>“Books are a uniquely portable magic.”<span>- Stephen King</span></h1>
    </header>
   )
}
 
export default Header
