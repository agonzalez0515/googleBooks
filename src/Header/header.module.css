.header {
  margin-top: 6rem;
}

.headerTitle {
  text-align: center;
  font-size: 3.5rem;
}

.headerTitle span {
  display: block;
  font-size: 1.6rem;
  float: right;
  margin-top: 1rem;
}