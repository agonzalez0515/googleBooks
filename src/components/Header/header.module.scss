@import '../../assets/mixins';

.header {
  margin-top: 6rem;
}

.headerTitle {
  text-align: center;
  font-size: 1.8rem;

  @include for-tablet-portrait-up {
    font-size: 3.5rem;
  }
}

.headerTitle span {
  display: block;
  font-size: 1rem;
  margin-top: 1rem;

  @include for-tablet-portrait-up {
    font-size: 1.6rem;
  }
}
