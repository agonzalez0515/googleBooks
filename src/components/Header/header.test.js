import React from 'react'
import { render } from '@testing-library/react'
import Header from './Header'

it('renders', () => {
  const { asFragment }= render(<Header />)
  expect(asFragment()).toMatchSnapshot()
})
  
