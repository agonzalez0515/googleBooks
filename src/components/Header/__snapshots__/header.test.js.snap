// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <header
    class="header"
  >
    <h1
      class="headerTitle"
    >
      Books are a uniquely portable magic.
      <span>
        - Stephen King
      </span>
    </h1>
  </header>
</DocumentFragment>
`;
